Source/JavaScriptCore/dfg/DFGCloneHelper.h

```diff
@@ -323,7 +324,7 @@ BasicBlock* CloneHelper::cloneBlock(BasicBlock* const block, const CustomizeSucc
     CLONE_STATUS(StringCharAt, Common) \
     CLONE_STATUS(StringCharCodeAt, Common) \
     CLONE_STATUS(StringCodePointAt, Common) \
-    CLONE_STATUS(StringFromCharCode, Common) \
+    /* CLONE_STATUS(StringFromCharCode, Common) // Not supported due to performance regression rdar://150526635 */ \
```

Comment:
I rather would like to keep this available in CloneHelper and disabling optimization in Unrolling phase side.

1. CloneHelper should be able to clone whatever the node is
2. LoopUnrolling phase should do analysis and giving up unrolling when `StringFromCharCode` node is found in this case.